### DISCUSSION TITLE
Where are these tutorials on equations?

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2347] [Q&A]
Hi Professor, I want to know what equation openpnm calculates, such as the three conservation equations in computational fluid dynamics. Then how to deal with them? Where are these tutorials on equations? I'm sorry to bother you again.

**Comment by jgostick** [comment]
The examples for doing various simulations are [here](https://github.com/PMEAL/OpenPNM/tree/dev/examples/notebooks/algorithms).  They don't go into the details of the equations, but they show how to use them.  The equations are well outlined in the literature, like the review by Jivkov, or any of our PNM papers.
